pom.xml

```diff
@@ -51,6 +51,17 @@
             <artifactId>github</artifactId>
             <version>1.14.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>branch-api</artifactId>
+            <version>0.2-beta-6</version>
```

Comment:
+1 for consistency, but requires cross-project updates.

However, it already knows specifically one caller. It's implemented specifically for one type of purpose. If something else calls it, no problem. That doesn't mean you must hide one well known (in fact, the only) integration point.